extension/content.js

```diff
@@ -430,6 +430,13 @@ function addTitleToEmojis() {
 	}
 }
 
+function sortMileStonesByClosestDueDate() {
+	const button = $('.subnav-item[href$="milestones"');
+	if (button) {
```

Comment:
jQuery returns an array even if it's not found so if array will always be true. Refer to #454 for the suggested implementation.